model_saver.py

```diff
@@ -0,0 +1,28 @@
+import joblib
+import logging
+
+
+class MovieSaver:
+    def __init__(self):
+        self.logger = self.setup_logger()
+
+    @staticmethod
+    def setup_logger():
```

Comment:
what are the reasons for setting up a logger and then using the global logging package instead of the configured logger?